puppet-checks/src/main/resources/org/sonar/l10n/pp/rules/puppet/S1066.html

```diff
@@ -0,0 +1,21 @@
+<p>
+    Merging collapsible <code>if</code> statements increases the code's readability.
+</p>
+
+<h2>Noncompliant Code Example</h2>
+
+<pre>
+if condition1 {
+    if condition2 {
```

Comment:
Could you indent with 2 whitespaces instead of 4 to be consistent with the other rule descriptions? I know I'm a pain in the a... :-)